src/xenia/kernel/kernel_state.cc

```diff
@@ -229,37 +229,53 @@ object_ref<XModule> KernelState::GetModule(const char* name, bool user_only) {
       return retain_object(user_module.get());
     }
   }
-  return nullptr;
+
+  // Module not found, try loading it
+  return LoadUserModule(name);
```

Comment:
We shouldn't implicitly load a module here - this function is called from places like XexGetModuleHandle that explicitly do not load new modules.

What we can do instead is in SetuplibraryImports, if `kernel_state_->IsKernelModule(name)` is false, call `GetModule`, and if no module is returned then implicitly load one.